### DISCUSSION TITLE
Install both IM6 and IM7 on a Mac?

### DISCUSSION THREAD

**Discussion by jtlocsei** [ImageMagick/ImageMagick#2183] [Help]
Is it possible to have IM6 and IM7 installed at the same time on a mac? How would I go about doing this? I currently have IM6 installed. I want to try out IM7 but I don't want to lose my IM6 installation that I'm already using.

**Comment by fmw42** [comment]
Contact me offline and I will try to help you. You can find my email address on my web site linked from my profile here.

**Reply by jtlocsei** [reply]
Thanks for your offer of help! In the meantime I realised that I could download the IM7 binary without messing up my homebrew installation of IM6.
